sqldelight-gradle-plugin/build.gradle

```diff
@@ -67,15 +67,18 @@ test {
       ":dialects:sqlite-3-38:publishAllPublicationsToInstallLocallyRepository",
       ":sqldelight-compiler:dialect:publishAllPublicationsToInstallLocallyRepository",
       ":runtime:runtime:publishKotlinMultiplatformPublicationToInstallLocallyRepository",
+      ":runtime:runtime:publishIosArm64PublicationToInstallLocallyRepository",
       ":runtime:runtime:publishJvmPublicationToInstallLocallyRepository",
       ":runtime:runtime:publishJsPublicationToInstallLocallyRepository",
       ":runtime:runtime-async:publishKotlinMultiplatformPublicationToInstallLocallyRepository",
       ":runtime:runtime-async:publishJvmPublicationToInstallLocallyRepository",
       ":runtime:runtime-async:publishJsPublicationToInstallLocallyRepository",
       ":runtime:internal:publishKotlinMultiplatformPublicationToInstallLocallyRepository",
+      ":runtime:internal:publishIosArm64PublicationToInstallLocallyRepository",
       ":runtime:internal:publishJvmPublicationToInstallLocallyRepository",
       ":runtime:internal:publishJsPublicationToInstallLocallyRepository",
       ":runtime:shared:publishKotlinMultiplatformPublicationToInstallLocallyRepository",
+      ":runtime:shared:publishIosArm64PublicationToInstallLocallyRepository",
```

Comment:
The changes in this file are unrelated and were required for some Gradle tests to pass on my mac. I'm assuming they pass on CI because they're run on linux and therefore the task are disabled.

Reply:
Oh well.. `publishIosArm64PublicationToInstallLocallyRepository` fails on linux. I reverted this part. Will look into it in a separate PR